plugins/permission-node/api-report.md

```diff
@@ -95,6 +95,22 @@ export const createPermissionIntegrationRouter: <TResource>(options: {
   getResource: (resourceRef: string) => Promise<TResource | undefined>;
 }) => Router;
 
+// @public
+export const createPermissionRule: <
```

Comment:
Does this actually need to exist? Not sure you would ever throw together a permission rule without having a full resource owner integration set up. It's ofc also just a utility so edge-cases are supported anyway

On the other hand it also makes a lot of sense to the point where it might be confusing to have `makeCreatePermissionRule` but not this

Reply:
Yeah, I think it's useful - if you've got checks elsewhere in your code that make sure the rules you're accepting have the right TResource and TQuery, you can skip the custom createXPermissionRule and just use this one and type inference to do the same job without enforcing consistency.